tests/buildtest.cmd

```diff
@@ -106,11 +106,9 @@ set "__CMakeBinDir=%__CMakeBinDir:\=/%"
 if not defined __CleanBuild goto SkipCleanBuild
 echo %__MsgPrefix%Doing a clean test build
 
-:: MSBuild projects would need a rebuild
-set __msbuildCleanBuildArgs=/t:rebuild
```

Comment:
Instead of removing the argument, have you looked into adding a `rebuild` target to the project?